### DISCUSSION TITLE
MonitorControl will not adjust the volume on my Samsung S80A Monitor

### DISCUSSION THREAD

**Discussion by Expertcoder111111** [MonitorControl/MonitorControl#1266] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

(I am connected to a 2021 MacBook Pro with the M1 Max chip) 
- After downloading MonitorControl, I was able to adjust brightness on my new S80A, and the sound controls on the keyboard were working.
- The volume controls on the keyboard, although working, weren't affecting the output volume of the monitor (which has built-in speakers)

### Expected behavior

When I play audio with the monitor selected as the audio output, the audio should play through the monitor speakers.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.0 Ventura
- Mac model: MacBook Pro (16-inch, 2021)
- MonitorControl version: v4.1.0
- Monitor(s):Samsung S80a
- Monitor Cable(s)/Connection(s):USB-c (MacBook Pro) to USB-c (Samsung S80a)
- Apple Silicon/M1 (yes or no):yes
```

**Comment by waydabber** [comment]
Unfortunately many monitors do not support volume controls via DDC. Several Samsung displays fall to this category. There is nothing much we can do about it I am afraid.
